src/test/java/io/confluent/kafkarest/unit/ConsumerManagerTest.java

```diff
@@ -172,7 +172,7 @@ public void testConsumerNormalOps() throws InterruptedException, ExecutionExcept
     String cid = consumerManager.createConsumer(
         groupName, new ConsumerInstanceConfig(EmbeddedFormat.BINARY));
     sawCallback = false;
-    consumerManager.readTopic(
+    Object result = consumerManager.readTopic(
```

Comment:
Nope, oversight from some debugging which I've removed in an updated patch. Thanks for catching that!